js/render-comments.js

```diff
@@ -0,0 +1,37 @@
+const commentsCurrentCount = document.querySelector('.comments-current-count');
+const commentsButton = document.querySelector('.comments-loader');
+const commentsBlock = document.querySelector('.social__comments');
+const commentTemplate = document.querySelector('#comment').content.querySelector('.social__comment');
+
+let shownComments = 0;
+
+const renderComments = (comments) => {
+	const commentFragment = document.createDocumentFragment();
+
+	comments.forEach((comment) => {
+		const commentElement = commentTemplate.cloneNode(true);
+		const commentAvatar = commentElement.querySelector('.social__picture');
+		commentAvatar.src = comment.avatar;
+		commentAvatar.alt = comment.name;
+		commentElement.querySelector('.social__text').textContent = comment.message;
+		commentFragment.append(commentElement);
+	});
+	commentsBlock.append(commentFragment);
+};
+
+export const addComments = (comments) => () => {
+	const remainingComments = comments.length - shownComments;
+	const newComments = Math.min(remainingComments, 5);
+	const commentsToShow = comments.slice(shownComments, shownComments + newComments);
+	renderComments(commentsToShow);
+	shownComments += newComments;
```

Comment:
Непонятно по названию, что создаётся функция. Так же непонятно, где comments в названии означает объект комментария, а где просто количество из-за того, что они названы похоже
```suggestion
export const createCommentRenderer = (comments) => () => {
	const remainingAmount = comments.length - shownComments;
	const newAmount = shownComments + Math.min(remainingAmount, 5);
	const commentsToShow = comments.slice(shownComments, newAmount);

	renderComments(commentsToShow);

	shownComments = newAmount;
```